fastapi/dependencies/utils.py

```diff
@@ -405,7 +433,7 @@ def is_coroutine_callable(call: Callable) -> bool:
     values.update(cookie_values)
     errors += path_errors + query_errors + header_errors + cookie_errors
     if dependant.body_params:
-        body_values, body_errors = await request_body_to_args(  # type: ignore # body_params checked above
```

Comment:
This `# type: ignore` is not necessary for mypy to pass; let me know if you want this in a separate PR